seodongheee2/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
+const userRouter = require('./userRouter');
+const postRouter = require('./postRouter');
+
+router.use('/users', userRouter.router);
+router.use('/users', postRouter.router);
```

Comment:
게시글에 대한 대분기가 users라는 엔드포인트로 이루어지네요!

```suggestion
router.use('/posts', postRouter.router);
```